### PR TITLE
Release/1.2.0

### PR DESCRIPTION
**Pull request by ventura-rivera**
1.2.0 is released! The biggest change to this small release contains a syntax feature that assigns `Scalar.minumum` and `Scalar.maximum` values. (See [PIF Scalar Object](https://citrineinformatics.github.io/pif-documentation/schema_definition/common/Scalar.html)). The features in this release have been tested by manually running test files on stage as per the [Testing and Deploying Ingesters](https://citrine.atlassian.net/wiki/spaces/PLA/pages/33161424/Testing+and+Deploying+Ingesters) documentation.

### What’s New

- You can now assign scalar minimum and maximum values using the following syntax in your csv files: `range(min, max)` where min and max will be assigned to `Scalar.minimum` and `Sacalar.maximum`, respectively. (#17)

### Improvements

- Added build status to README (#5)
- Improving test checks on headers (#16)
- A `test_requirements.txt` was created and includes pytest (#17)

### Bugfixes

- Python versions 3.3 and 3.4 were removed from the test set in `.travis.yml`. (#18)
- Ensuring empty subsystems are not assigned to the main pif system (#12)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='csv_template_ingester',
-    version='1.1.0',
+    version='1.1.1',
```
Comment (maxhutch) [inline setup.py]: This is a new feature, so it should be a minor version bump.  See [semver](https://semver.org/)

Comment (ventura-rivera) [reply setup.py]: rgr, I'll get on this

**Comment by ventura-rivera** [comment]
Excuse me closing this branch. I was trying to rename it locally and remotely, and thought this PR would remain open. I am just going to use the GUI to rename it.

**Comment by ventura-rivera** [comment]
I have tested on stage, and the test files have successfully ingested. This PR is ready to be merged.

**Pull request merged**